components/Navigation/Navigation.tsx

```diff
@@ -79,13 +81,13 @@ export const Navigation = () => {
         {config.map((link, index) => (
           <ItemNavigation {...link} key={index} />
         ))}
-        <form onSubmit={handleSubmit}>
-          <Button
-            type="submit"
-            text="Logout"
-            variant={ButtonVariants.primary}
-          />
-        </form>
+        <ButtonWrapperSC>
+          <form onSubmit={handleSubmit}>
```

Comment:
hmm, form with just 1 button? Strange form, i suggest add just Log out click handler and do all needed logic